src/rest.ts

```diff
@@ -1,18 +1,23 @@
 import cockpit from "cockpit";
 
-import { debug } from "./util.js";
+import { debug } from "./util.tsx";
 
-function manage_error(reject, error, content) {
-    let content_o = {};
-    if (content) {
+type JsonObject = cockpit.JsonObject;
+
+// make this `unknown` to conveniently call it on raw error objects
+function format_error(error: object, content: unknown): object {
+    let content_o: JsonObject = {};
+    if (typeof content === 'string') {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2214-a54e447f-20250801-060648-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)